pulp_container/app/tasks/sync_stages.py

```diff
@@ -327,9 +331,7 @@ def create_tagged_manifest_list(self, tag_name, saved_artifact, manifest_list_da
         """
         digest = f"sha256:{saved_artifact.sha256}"
         manifest_list = Manifest(
-            digest=digest,
-            schema_version=manifest_list_data["schemaVersion"],
-            media_type=manifest_list_data["mediaType"],
+            digest=digest, schema_version=manifest_list_data["schemaVersion"], media_type=media_type
```

Comment:
for the case when `media_type` won't still be available, you should decide what type of manifests list is this based on one of the elements from the `manifests` `mediatype`.
These are required fields.

Reply:
I made the media_type variable "aware" of the `manifests` field. Then, it makes sense to use it in this way. What do you think?